zerver/views/__init__.py

```diff
@@ -764,7 +773,7 @@ def create_realm(request, creation_key=None):
                               request=request)
 
 def confirmation_key(request):
-    return json_success(request.session.get('confirmation_key'))
+    return json_success({'confirmation_key': os.getenv('CONFIRMATION_KEY')})
```

Comment:
OK; I suspect that this reflects an actual bug in how your code is storing the confirmation keys in the session objects that we should fix; this solution won't work in production since the production system is run multi-process and the OS environment thus isn't a valid way to share state from one request to another.  The Django session is, so we shoudl use that.